tests/e2e/specs/all-fields.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * External dependencies
+ */
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import { v4 as uuid } from 'uuid';
+import { getDocument, queries } from 'pptr-testing-library';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	clickButton,
+	createNewPost,
+	insertBlock,
+	visitAdminPage,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'AllFields', () => {
+	it( 'creates the block and makes the fields available in the block editor', async () => {
```

Comment:
This e2e test is much more complete than before.

It used to just test a text field.

Now, it tests all 13 free fields. 

1. Creates a GCB block
2. Adds one of each to the block
3. Creates a post
4. Adds a value for each field
5. Ensures the `<ServerSideRender>` has the right value for each field